apps/backoffice-v2/src/common/hooks/useHomeLogic/useHomeLogic.ts

```diff
@@ -146,26 +165,29 @@ export const useHomeLogic = () => {
 
   const casesByStatus = useMemo(() => {
     if (!currentStats) return [];
-    return currentStats.casesByStatus.map(item => ({
-      ...item,
-      href: `/${locale}/case-management?filter[status][0]=${item.status}`,
-    }));
+    return currentStats.casesByStatus;
+    // return currentStats.casesByStatus.map(item => ({
+    //   ...item,
+    //   href: `/${locale}/case-management?filter[status][0]=${item.status}`,
+    // }));
   }, [currentStats]);
 
   const ongoingCasesByRisk = useMemo(() => {
     if (!currentStats) return [];
-    return currentStats.ongoingCasesByRisk.map(item => ({
-      ...item,
-      href: `/${locale}/case-management?filter[status][0]=active&filter[riskLevel][0]=${item.riskLevel}`,
-    }));
+    return currentStats.ongoingCasesByRisk;
+    // return currentStats.ongoingCasesByRisk.map(item => ({
+    //   ...item,
+    //   href: `/${locale}/case-management?filter[status][0]=active&filter[riskLevel][0]=${item.riskLevel}`,
+    // }));
   }, [currentStats]);
 
   const approvedCasesByRisk = useMemo(() => {
     if (!currentStats) return [];
-    return currentStats.approvedCasesByRisk.map(item => ({
-      ...item,
-      href: `/${locale}/case-management?filter[status][0]=completed&filter[riskLevel][0]=${item.riskLevel}`,
-    }));
+    return currentStats.approvedCasesByRisk;
+    // return currentStats.approvedCasesByRisk.map(item => ({
+    //   ...item,
+    //   href: `/${locale}/case-management?filter[status][0]=completed&filter[riskLevel][0]=${item.riskLevel}`,
+    // }));
```

Comment:
```suggestion
    return currentStats.approvedCasesByRisk;
```
can we discard commented-out code?

Reply:
we can, i left it since we will likely need to implement this feature in the future.
i also left the code in the pie charts that handles passed `href` props accordingly.